src/Components/FormCreation/Submission/CreateSubmissionForm.js

```diff
@@ -0,0 +1,79 @@
+import React, { useReducer, useEffect, useState, useContext } from "react";
+import styled from "styled-components";
+import FormSection from "./FormSection";
+import { AuthContext } from "../../../Authentication/Auth.js";
+import * as FORM from "../../../requests/forms.js";
+import usePromise from "../../../Hooks/usePromise";
+import CreateEditFormHeader from "./SubmissionFormHeader";
+import { defaultFormState } from "./SubmissionFormStateManagement";
+import customFormSectionsReducer from "../../../Reducers/CustomFormSectionsReducer";
+
+const FormWrapper = styled.div`
+  padding-top: 70px;
+  padding-left: 15%;
+`;
+
+function CreateSubmissionForm() {
+  const { appUser } = useContext(AuthContext);
+  const [sections, dispatchSectionsUpdate] = useReducer(
+    customFormSectionsReducer,
+    []
+  );
+  const [activeSection, setActiveSection] = useState(0);
+  const [loadForm, refetch] = usePromise(FORM.getForm, {
+    programId: appUser.currentProgram
+  });
+  const [headerData, setHeaderData] = useState({
```

Comment:
We shouldn't need this for formSubmission right?

Reply:
That is right, I can remove